02/Elena Davydik/src/components/Main/HotelsSearchForm/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { ChangeEvent } from "react";
+
+type Props = {
+    onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+}
+
+const searchFormTitle: string = 'Available hotels';
```

Comment:
you can skip type declaration for primitives

Reply:
Skip type declaration for variable _searchFormTitle_